src/drawer.js

```diff
@@ -686,8 +688,9 @@ $.Drawer.prototype = {
         var pixelDensityRatio = $.pixelDensityRatio;
         var viewportSize = this.viewport.getContainerSize();
         return {
-            x: viewportSize.x * pixelDensityRatio,
-            y: viewportSize.y * pixelDensityRatio
+            // canvas width and height are integers
+            x: Math.floor(viewportSize.x * pixelDensityRatio),
```

Comment:
It's probably not a big deal, but I think we might as well `round` as it'll be slightly more accurate, and I don't think there's any need to stick to floor in this case.

Reply:
Sure.
I used floor to keep the keep the old behavior (truncating non-integer into integer), but round is probably better and we should "keep" wrong behaviors.